README.md

```diff
@@ -11,6 +11,10 @@
 
 By using the kubernetes [vertical-pod-autoscaler](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler) in recommendation mode, we can see a suggestion for resource requests on each of our apps. This tool creates a VPA for each deployment in a namespace and then queries them for information.
 
+<div align="center">
+<img src="img/screenshot.png" alt="Goldilocks Screenshot" />
+<br>
+
```

Comment:
I just viewed this file and it looks like this addition made everything below it also be center aligned. Does the <div> need to be closed?

Reply:
Whoops good catch! Thanks.

Here's what the end result looks like: https://github.com/FairwindsOps/goldilocks/tree/rb/screenshot

Reply:
Yay! It looks great!! ✨